### PR TITLE
Switching fill fields to golden patient

### LINKED ISSUE #2646: Align patient selection between design and dev

### PR DESCRIPTION
**Pull request by robertandremitchell**
# PULL REQUEST

## Summary
* Adds Hyper Unlucky as the fill field patient for most* demo cases
* Adds Public HAPI: Direct connections
* Makes Public HAPI: Direct connection the default FHIR Server
* Standardizes URL formation (Patient originally did not start with `/`, which led to some funkiness when building FHIR URLs)

## Related Issue
Fixes #2646

## Acceptance Criteria
Please copy the acceptance criteria from your ticket and paste it here for your reviewer(s)

## Additional Information
* How do we want to handle the standard Newborn? For example, if we were to switch to Hyper Unlucky for everything, as is done for the other cases, what happens is that all the newborn screening data is returned. Do we need to create at least 1 other distinct patient to show a pass/referall/fail? Or update Hyper Unlucky to only have one newborn screening, then just have Mango/Watermelon/Cucumber in our back pocket in case we need to show a pass/referral/fail?
* How do we want to handle the `socialHistoryQuery` in `CustomQuery.ts`? In our other patient data, we did not have social history data, so to mimic existing demo displays, I commented it out so that the social determinants of health were not present. Is it better to display them?

Unrelated, but having some trouble with the e2e tests. If I run them individually, they pass. If I replicate them, they pass. But running the full script, 4 fail. Running on GitHub Actions, 6 fail. very confusing.

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/tefca-viewer/src/app/CustomQuery.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export class CustomQuery {
       this.diagnosticReportQuery,
       this.conditionQuery,
       this.medicationRequestQuery,
-      this.socialHistoryQuery,
+      // this.socialHistoryQuery,
```
Comment (robertandremitchell) [inline containers/tefca-viewer/src/app/CustomQuery.ts:101]: If we comment this back in, social determinants of health will populate for all of Hyper Unlucky's data. would this be okay?

File: containers/tefca-viewer/src/app/fhir-servers.ts [hunk 2]
```diff
@@ -14,20 +14,23 @@ type FHIR_SERVER_CONFIG = {
...
   "HELIOS Meld: eHealthExchange": configureEHX("MeldOpen"),
   "JMC Meld: Direct": {
-    hostname: "https://gw.interop.community/JMCHeliosSTISandbox/open/",
+    hostname: "https://gw.interop.community/JMCHeliosSTISandbox/open",
     init: {} as RequestInit,
   },
   "JMC Meld: eHealthExchange": configureEHX("JMCHelios"),
-  "Public HAPI: eHealthExchange": configureEHX("PublicHAPI"),
+  "Public HAPI: Direct": {
+    hostname: "https://hapi.fhir.org/baseR4",
```
Comment (robertandremitchell) [inline containers/tefca-viewer/src/app/fhir-servers.ts:27]: kind of just made the choice to remove the `/` here and add it to the URLs built in `query-service.ts` and `CustomQuery.ts`, but we can technically go the other way relatively easily. Not sure there's a standard convention / preferred method.

**Review by DanPaseltiner — APPROVED** [review]
🚀

**Comment by robertandremitchell** [comment]
@m-goggins , I'm now consistently able to get tests to pass locally consistently, even from a completely clean cache. do you have any ideas for what could be going wrong on GitHub Actions?

**Pull request closed without merging**